src/OrbitBase/include/OrbitBase/NotFoundOr.h

```diff
@@ -39,6 +39,13 @@ template <typename T>
   return std::get<NotFound>(not_found_or).message;
 }
 
+// Free function to get the "found" content of a NotFoundOr object.
+template <typename T>
+[[nodiscard]] T GetFound(const std::variant<T, NotFound>& not_found_or) {
```

Comment:
We get into templates so I don't know. Doesn't this make a copy of the content? And what about move-only types? Shouldn't we return by const-ref? Or is it already done?

Reply:
Its good that you call me out on that. The same idea about const ref popped into my head, but I was to lazy to think it through. Now I did that and I think returning a const ref cannot lead to any problems, because the caller owns the original object. I will change it.

Reply:
std::get returns a const ref as well.

Reply:
After looking at std::get, I decided to also add a move overload.

Reply:
Nice. So as a last question while we are here, does it make sense for `GetNotFoundMessage` to also have such an overload?

Reply:
Yes I added an overload. This can be used for better performance (not copying the string). I need to rebase, because of merge conflicts in OrbitApp. If you still want to look at the change (its pretty trivial).. here is a screenshot: https://screenshot.googleplex.com/5Np3VmmGTz22mfP

I will merge after the rebase

Reply:
Thanks for tackling this.